src/main/kotlin/tk/dungeonlurkers/cryptr/configs/SecurityConfiguration.kt

```diff
@@ -0,0 +1,22 @@
+package tk.dungeonlurkers.cryptr.configs
```

Comment:
IMO `src/main/kotlin/tk/` is too long. What was the reasons for that?

Reply:
```src/main/kotlin``` is autogenerated by Gradle
```tk/dungeonlurkers``` is proper package naming in Java ecosystem